tools/lib-alert-tree/lib_alert_tree/kubernetes.py

```diff
@@ -3,14 +3,24 @@
 from . import models
 
 
+def container_alerts(name, container=".*", severity="warning", namespace="default"):
```

Comment:
I would set this the otherway around :) 
name for container and another arg for the pod name

Reply:
I'll do that, but I prefer that way because I think more readable.

Reply:
Why do you think it's more readable ?
To me it's less readable, here you have a function `container_alerts` with a `name` argument that is not the container name but a pod name, so to me, it's not super clear